conf/security_events.conf.defaults

```diff
@@ -222,7 +242,7 @@ actions=enforce_provisioning,log
 template=enforce_provisioning
 trigger=Provisioner::check
 enabled=Y
-grace=15m
+grace=0s
```

Comment:
Its not a good idea to have a grace of 0s on this, lets put the 15m back or at least 1m.

Otherwise if it gets triggered twice in a connection, it results in 2 checks